lms/djangoapps/certificates/models.py

```diff
@@ -132,6 +132,12 @@ class Meta:
     allowlist = models.BooleanField(default=0)
     notes = models.TextField(default=None, null=True)
 
+    # This is necessary because CMS does not install the certificates app, but it
```

Comment:
Good call on adding history to the new model, I just realized the old one didn't have it. 👍